packages/playground/src/domains/agent-builder/hooks/__tests__/use-stored-agent-favorite.editor.msw.test.tsx

```
@@ -0,0 +1,65 @@
+import { act, renderHook, waitFor } from '@testing-library/react';
+import { http, HttpResponse } from 'msw';
+import { describe, expect, it } from 'vitest';
+
+import { useToggleStoredAgentFavorite } from '../use-stored-agent-favorite';
+import { favoritedAgent, makeStoredAgent, makeStoredAgentsList } from '../../../agents/hooks/__tests__/fixtures/editor-agents';
```

ESLint

[error] `../../../agents/hooks/__tests__/fixtures/editor-agents` import should occur before import of `../use-stored-agent-favorite`

(import/order)